src/providers/golangCompletion.ts

```diff
@@ -55,6 +55,7 @@ export class GolangProvider implements ApiProvider {
     apiType: 'call_api' | 'call_embedding_api' | 'call_classification_api',
   ): Promise<any> {
     const absPath = path.resolve(path.join(this.options?.config.basePath || '', this.scriptPath));
+    const moduleRoot = path.dirname(absPath); // Assuming the go.mod is in the same directory as the script
```

Comment:
hi @mldangelo , I believe this line has broken our evaluation script since our go.mod file is in a parent directory of the main.go file used to run promptfoo evaluations. I have raised an issue here: https://github.com/promptfoo/promptfoo/issues/3057

Thanks!